spec/app_spec.rb

```diff
@@ -1,174 +1,174 @@
 require 'spec_helper'
 
-describe "app" do
-
-  describe "access control" do
+describe 'app' do
+  describe 'access control' do
     let(:user) { create_test_user(42) }
     # all routes (even nonexistent ones) are covered by the api key
     # /heartbeat is the only exception, covered in the heartbeat tests below
-    let(:urls) { {
-        "/" => 404,
-        "/api/v1/users/#{user.id}" => 200,
-        "/api/v1/users/doesnotexist" => 404,
-        "/selftest" => 200
+    let(:urls) do
+      {
+          '/' => 404,
+          "/api/v1/users/#{user.id}" => 200,
+          '/api/v1/users/doesnotexist' => 404,
+          '/selftest' => 200
       }
-    }
+    end
 
-    it "returns 401 when api key header is unset" do
-      urls.each do |url, _|
+    it 'returns 401 when api key header is not set' do
+      urls.keys.each do |url|
         get url
-        last_response.status.should == 401
+        expect(last_response.status).to eq 401
       end
     end
 
-    it "returns 401 when api key value is incorrect" do
-      urls.each do |url, _|
-        get url, {}, {"HTTP_X_EDX_API_KEY" => "incorrect-#{TEST_API_KEY}"}
-        last_response.status.should == 401
+    it 'returns 401 when api key value is incorrect' do
+      urls.keys.each do |url|
+        get url, {}, {'HTTP_X_EDX_API_KEY' => "incorrect-#{TEST_API_KEY}"}
+        expect(last_response.status).to eq 401
       end
     end
 
-    it "allows requests when api key value is correct" do
+    it 'allows requests when api key value is correct' do
       urls.each do |url, status|
-        get url, {}, {"HTTP_X_EDX_API_KEY" => TEST_API_KEY}
-        last_response.status.should == status
+        get url, {}, {'HTTP_X_EDX_API_KEY' => TEST_API_KEY}
+        expect(last_response.status).to eq status
       end
     end
   end
 
-  describe "heartbeat monitoring" do
-    it "does not require api key" do
-      get "/heartbeat"
-      last_response.status.should == 200
+  describe 'heartbeat monitoring' do
+    subject do
+      get '/heartbeat'
+      last_response
+    end
+
+    it 'does not require api key' do
+      get '/heartbeat'
```

Comment:
@clintonb: I think this is handled as part of subject, so I should remove.  Correct?